recipes/fmpy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "fmpy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
```

Comment:
Please install from a tarball instead of git:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

Reply:
Ok, thanks for the hint. In my company, we typically build everything from Git repos as we are behind a firewall and downloading from PyPI doesn't work, but I have understood that cloning every repo is not reasonable for something big like conda-forge. The most recent version on PyPI uses wheels, for which I am not sure how the recipe should look like. I'll use the tarball from GitHub instead if that is ok.